### PR TITLE
replacing dev setup section with quick setup vagrant example

### PR DESCRIPTION
**Pull request by heckj**

### CODE REVIEW

File: docs/development_guide.rst [hunk 1]
```diff
@@ -3,27 +3,60 @@ Development Guide
...
+   `cd ~`
+   `git clone https://github.com/rackhd/on-imagebuilder`
+   `cd on-imagebuilder`
+   `sudo ansible-playbook -i hosts all.yml`
 
-**Checking Out the Code**
+5. Move the resulting files into the HTTP server's static files directory
 
-1. Clone at minimum the **on-http**, **on-dhcp-proxy**, **on-tftp**, **on-taskgraph**, and **on-tasks** repositories.
+   `mv /tmp/on-imagebuilder/builds/* ~/RackHD/on-http/static/http/common`
```
Comment (benbp) [inline docs/development_guide.rst]: As @DavidjohnBlodgett pointed out in the other PR, change the second path to `~/src/on-http/static/http/common/`

File: docs/development_guide.rst [hunk 2]
```diff
@@ -3,27 +3,60 @@ Development Guide
...
+
+
+1. Clone at minimum the rackHD repository. This repository includes both git
+   submodules with all the relevant source code, as well as an example vagrant
+   instance to get an instance up and running locally very easily.
+
+   `git clone https://github.com/RackHD/RackHD`
+
+   Within the example/ directory is a setup script to set up an "example rack"
+   entirely virtually - albiet a very simple one. It creates a virtualbox with
```
Comment (DavidjohnBlodgett) [inline docs/development_guide.rst]: should this be "entirely virtual" ? @benbp

File: docs/development_guide.rst (hunk 1 above)
Comment (DavidjohnBlodgett) [inline docs/development_guide.rst]: yep^^

File: docs/development_guide.rst (hunk 2 above)
Comment (heckj) [inline docs/development_guide.rst]: damned spelling nazis

Comment (benbp) [inline docs/development_guide.rst]: You also misspelled `albeit`

Comment (benbp) [inline docs/development_guide.rst]: Also creates a virtualbox isn't correct, probably "creates a virtualbox VM"?

Comment (benbp) [inline docs/development_guide.rst]: And entirely virtually I think *is* actually correct, but it probably sounds better to just say "virtually" and strike "entirely"

**Comment by DavidjohnBlodgett** [comment]
In addition, does the directions leave the user in the image-builder directory right before asking them to  run "sudo nf start"? because that command must be run from ~/ (aka /home/vagrant) due to needing the forman proc file @benbp @heckj

**Comment by benbp** [comment]
Good point.

**Comment by DavidjohnBlodgett** [comment]
:+1:

File: docs/development_guide.rst [hunk 3]
```diff
@@ -3,27 +3,61 @@ Development Guide
...
+The example setup uses Vagrant and ansible locally. Please install them
+directly on your local machine.
+
+
+
+1. Clone at minimum the rackHD repository. This repository includes both git
+   submodules with all the relevant source code, as well as an example vagrant
+   instance to get an instance up and running locally very easily.
+
+   `git clone https://github.com/RackHD/RackHD`
```
Comment (clintonskitson) [inline docs/development_guide.rst]: Should this be `--recursive`?

**Comment by heckj** [comment]
verified baseline setup with rackhd/example vagrant structure = took ~70 minutes on a home machine (macbook air) with nothing cached and a DSL line. Quite slow

**Pull request merged**